src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,48 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.io.IOException;
```

Comment:
Perhaps the imports could be in a uniform sequence across all files to make the code cleaner